airbyte/helm/airbyte/values.yaml

```diff
@@ -45,131 +45,152 @@ private:
+  - name: worker-mem
+    labels:
+      platform.plural.sh/component: airbyte-worker
+      platform.plural.sh/kind: deployment
+      platform.plural.sh/resource: memory
+    spec:
+      name: worker memory
+      documentation: memory requests for worker deployment
+      updates:
+        - path:
+            ["airbyte", "airbyte", "worker", "resources", "requests", "memory"]
+  - name: scheduler-cpu
+    labels:
+      platform.plural.sh/component: airbyte-scheduler
+      platform.plural.sh/kind: deployment
+      platform.plural.sh/resource: cpu
+    spec:
+      name: scheduler cpu
+      documentation: cpu requests for scheduler deployment
+      updates:
+        - path:
+            ["airbyte", "airbyte", "scheduler", "resources", "requests", "cpu"]
+  - name: scheduler-mem
+    labels:
+      platform.plural.sh/component: airbyte-scheduler
+      platform.plural.sh/kind: deployment
+      platform.plural.sh/resource: memory
+    spec:
+      name: scheduler memory
+      documentation: memory requests for scheduler deployment
+      updates:
+        - path:
+            [
+              "airbyte",
+              "airbyte",
+              "scheduler",
+              "resources",
+              "requests",
+              "memory",
+            ]
+  - name: server-cpu
+    labels:
+      platform.plural.sh/component: airbyte-server
+      platform.plural.sh/kind: deployment
+      platform.plural.sh/resource: cpu
+    spec:
+      name: server cpu
+      documentation: cpu requests for server deployment
+      updates:
+        - path: ["airbyte", "airbyte", "server", "resources", "requests", "cpu"]
+  - name: server-mem
+    labels:
+      platform.plural.sh/component: airbyte-server
+      platform.plural.sh/kind: deployment
+      platform.plural.sh/resource: memory
+    spec:
+      name: server memory
+      documentation: memory requests for server deployment
+      updates:
+        - path:
+            ["airbyte", "airbyte", "server", "resources", "requests", "memory"]
+  - name: temporal-cpu
+    labels:
+      platform.plural.sh/component: airbyte-temporal
+      platform.plural.sh/kind: deployment
+      platform.plural.sh/resource: cpu
+    spec:
+      name: temporal cpu
+      documentation: cpu requests for temporal deployment
+      updates:
+        - path:
+            ["airbyte", "airbyte", "temporal", "resources", "requests", "cpu"]
+  - name: temporal-mem
+    labels:
+      platform.plural.sh/component: airbyte-temporal
+      platform.plural.sh/kind: deployment
+      platform.plural.sh/resource: memory
+    spec:
+      name: temporal memory
+      documentation: memory requests for temporal deployment
+      updates:
+        - path:
+            [
+              "airbyte",
+              "airbyte",
+              "temporal",
+              "resources",
+              "requests",
+              "memory",
+            ]
 
 airbyte:
   postgresql:
-    enabled: false
+    enabled: true
```

Comment:
this is a bit of a pain, but this actually has a side-effect of causing the bitnami postgres helm chart to also be deployed, which we'd definitely prefer to avoid in preference of zalando's pg operator.  Perhaps we could use a different helm value to toggle this, or move it to the top-level postgres field?

Reply:
ah fair point, I did not pay attention
I will fix that

Reply:
yeah that looks like the right approach to me!